css/optimsoc.css

```diff
@@ -200,35 +219,60 @@ ul.share-buttons li {
   }
 }
 
-.optimsoc-doc-mainnav li div.active {
+/* Highlight current chapter */
+/* Latex: */ .optimsoc-doc-mainnav li div.active,
+/* Sphinx: */ .optimsoc-doc-mainnav .toctree-l1 .current {
   color: #F04C5C;
   font-weight: bold;
 }
 
-.optimsoc-doc-subnav {
+/* Sphinx only: don't have link for current */
+.optimsoc-doc-mainnav .toctree-l1 .current {
+   pointer-events: none;
+   cursor: default;
+}
+
+/* Current chapter subnav*/
+/* Latex: */ .optimsoc-doc-subnav,
+/* Sphinx: */ .optimsoc-doc-mainnav .toctree-l1 > ul
+{
   background-color: #FFFFFF;
   margin-bottom: 5px;
   margin-top: 5px;
   padding-bottom: 15px;
   padding-top: 15px;
 }
 
-.optimsoc-doc-subnav .nav > li > a {
+/* Sphinx only: This is the subnav */
+.optimsoc-doc-mainnav .toctree-l1 > ul
+{
+  padding-left: 0px;
+  padding-right: 0px;
+}
+
+/* Latex: */.optimsoc-doc-subnav .nav > li > a,
+/* Sphinx: */.optimsoc-doc-mainnav .toctree-l1 > ul > li > a
+{
   display: block;
   padding: 4px 20px;
 }
-.optimsoc-doc-subnav .nav > li > a:hover,
-.optimsoc-docs-sidebar .nav > li > a:focus {
+/* Latex: */.optimsoc-doc-subnav .nav > li > a:hover,
+.optimsoc-docs-sidebar .nav > li > a:focus,
+/* Sphinx: */.optimsoc-doc-mainnav ul > li > a:hover,
+.optimsoc-doc-mainnav ul > li > a:focus
+{
   text-decoration: none;
 }
 .optimsoc-doc-subnav .nav > .active > a,
 .optimsoc-docs-sidebar .nav > .active:hover > a,
-.optimsoc-docs-sidebar .nav > .active:focus > a {
+.optimsoc-docs-sidebar .nav > .active:focus > a
```

Comment:
Ah, I see. Let's revisit it after I finished some more reference manual documentation (it may meanwhile be tested using `build-docs.sh /path/to/optimsoc/sources`). I get the impression that this will be a bit harder when not messing with the html code generation in Sphinx.